clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/provider/agent/KubernetesServerGroupCachingAgent.groovy

```diff
@@ -110,8 +114,17 @@ class KubernetesServerGroupCachingAgent extends KubernetesCachingAgent implement
       loadReplicaSet(namespace, serverGroupName)
     }
 
+    V1beta1DaemonSet daemonSet = metricsSupport.readData {
```

Comment:
@lwander  -     is this about this line : 

    List<KubernetesController> serverGroups = (replicationControllerList.collect {
      it ? new KubernetesController(controller: it) : null
    } + replicaSetList.collect {
      it ? new KubernetesController(controller: it) : null
    } +   daemonSetList?.items.collect { it ? new KubernetesController(controller: it): null
    } + statefulSetList?.items.collect { it ? new KubernetesController(controller: it): null
    })- null

we should  segregate the above  in seperate variable ?